### PR TITLE
Refactored PersonalInformation 

### PR DESCRIPTION
**Pull request by Prince0906**
Ref: #4460 
### **About This PR**
This PR refactored 

> packages/volto/src/components/manage/Preferences/PersonalInformation.jsx

 from class based to functional component.

### CODE REVIEW

File: packages/volto/news/6177.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Refactor the PersonalInformation component from a class component to a functional component. @Prince0906
```
Comment (stevepiercy) [inline packages/volto/news/6177.feature:1]:
```suggestion
Refactor the `PersonalInformation` component from a class component to a functional component. @Prince0906
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor MyST syntax improvement.

**Review by stevepiercy — APPROVED** [review]
I approve only the news item. This PR needs a review from a @plone/volto-team member.

**Comment by ichim-david** [comment]
@Prince0906 before you attempt to port the class based components to functional components you need to look also at the opened pull requests that tackle this task, in particular the ones opened by https://github.com/plone/volto/pulls/Tishasoumya-02
This is because the information from the issue is not fully up to date on what pull requests are opened
https://github.com/plone/volto/issues/4460

Doing so will ensure that you don't work for nothing doing duplicated work.

**Comment by davisagli** [comment]
Thanks for your contribution, but I will close this PR because it is a duplicate of #5046 which was already in progress.

**Pull request closed without merging**